README.md

```diff
@@ -31,10 +31,16 @@ then start it with `docker-compose up`
 
```

Comment:
here's where everything starts to go off the rails and I get weird error messages. Everything else worked fine.

Can you make the changes I suggested, and then once I get some help on the problems I was experiencing, I will continue to work through these instructions.

Reply:
Yes, I will get these changes made.  Thanks for the feedback.